src/components/Coaching/CoachingDetail/ActivitySummary/ActivitySummary.test.tsx

```diff
@@ -1,158 +1,117 @@
+import { activitySummaryMocks } from '../coachingMocks';
 import { ActivitySummary } from './ActivitySummary';
 
-const mocks = {
-  ActivitySummary: {
-    reportsActivityResults: {
-      periods: [
-        {
-          startDate: '2023-09-01',
-          callsWithAppointmentNext: 1,
-          completedCall: 2,
-          completedPreCallLetter: 3,
-          completedReminderLetter: 4,
-          completedSupportLetter: 5,
-          completedThank: 6,
-          dials: 77,
-          electronicMessageSent: 8,
-          electronicMessageWithAppointmentNext: 9,
-        },
-        {
-          startDate: '2023-10-01',
-          callsWithAppointmentNext: 11,
-          completedCall: 12,
-          completedPreCallLetter: 13,
-          completedReminderLetter: 14,
-          completedSupportLetter: 15,
-          completedThank: 16,
-          dials: 97,
-          electronicMessageSent: 18,
-          electronicMessageWithAppointmentNext: 19,
-        },
-        {
-          startDate: '2023-11-01',
-          callsWithAppointmentNext: 31,
-          completedCall: 32,
-          completedPreCallLetter: 33,
-          completedReminderLetter: 34,
-          completedSupportLetter: 35,
-          completedThank: 36,
-          dials: 107,
-          electronicMessageSent: 38,
-          electronicMessageWithAppointmentNext: 39,
-        },
-      ],
-    },
-  },
-};
-
 const mutationSpy = jest.fn();
 
 describe('ActivitySummary', () => {
   it('renders the table data', async () => {
     const { findByRole, getAllByRole } = render(
-      <GqlMockedProvider mocks={mocks}>
+      <GqlMockedProvider
+        mocks={{
+          ActivitySummary: activitySummaryMocks,
+          LoadConstants: loadConstantsMockData,
+        }}
+      >
         <ActivitySummary
           accountListId="account-list-1"
-          period={CoachingPeriodEnum.Weekly}
+          period={CoachingPeriodEnum.Monthly}
         />
       </GqlMockedProvider>,
     );
 
     expect(
-      await findByRole('cell', { name: 'Phone Dials' }),
+      await findByRole('cell', { name: 'New Connections Added' }),
     ).toBeInTheDocument();
 
     const headers = getAllByRole('rowheader');
-    const phoneRow = headers[0];
-    expect(phoneRow.children[0]).toHaveTextContent('Phone Dials');
-    expect(phoneRow.children[1]).toHaveTextContent('Sep 1');
-    expect(phoneRow.children[2]).toHaveTextContent('Oct 1');
-    expect(phoneRow.children[3]).toHaveTextContent('Nov 1');
-    expect(phoneRow.children[4]).toHaveTextContent('Average');
-    expect(headers[1]).toHaveTextContent('Electronic Messages');
-    expect(headers[2]).toHaveTextContent('Correspondence');
+    const dateRow = headers[0];
+    expect(dateRow.children[0]).toHaveTextContent('');
+    expect(dateRow.children[1].textContent).toContain('Mar');
+    expect(dateRow.children[2].textContent).toContain('Apr');
+    expect(dateRow.children[3].textContent).toContain('May');
```

Comment:
For the sake of consistency.

```suggestion
    expect(dateRow.children[1]).toHaveTextContent('Mar');
    expect(dateRow.children[2]).toHaveTextContent('Apr');
    expect(dateRow.children[3]).toHaveTextContent('May');
```